bin/seeds.js

```diff
@@ -0,0 +1,104 @@
+// get mongoose
+const mongoose = require('mongoose');
+//access to book schema
+const Movie = require('../models/Movie.model');
+const DB_NAME = 'express-cinema-dev';
+
+//connect to db
+mongoose.connect(`mongodb://localhost/${DB_NAME}`, {
```

Comment:
The connection string to your database should be in the .env file (place this file in the root of the project)